.gitignore

```diff
@@ -1,11 +1 @@
-npm-debug.log*
-node_modules
-/build/
-/public/app.css
-.idea/
-yarn-error.log
-/coverage/
 .DS_Store
```

Comment:
I think it is better to have all required items in .gitignore

Reply:
good point, will do